liionsden/settings/settings.py

```diff
@@ -172,11 +176,9 @@
 # File storage
 # Azure blob storage for media files
 DEFAULT_FILE_STORAGE = "storages.backends.azure_storage.AzureStorage"
-AZURE_CONTAINER = "media"
-AZURE_ACCOUNT_NAME = "liionsdenmedia"
-AZURE_ACCOUNT_KEY = os.getenv("AZURE_STORAGE_ACCOUNT_KEY")
-AZURE_CUSTOM_DOMAIN = f"{AZURE_ACCOUNT_NAME}.blob.core.windows.net"
-MEDIA_URL = f"https://{AZURE_CUSTOM_DOMAIN}/{AZURE_CONTAINER}/"
+AZURE_CONTAINER = os.getenv("AZURE_STORAGE_CONTAINER")
+AZURE_CONNECTION_STRING = os.getenv("AZURE_STORAGE_CONNECTION_STRING")
+AZURE_ACCOUNT_NAME = os.getenv("AZURE_STORAGE_ACCOUNT_NAME")
```

Comment:
Could get this from the connection string via parse_connection_string.

Reply:
I did try this and for whatever reason it doesn't work. Replacing L181 with: `AZURE_ACCOUNT_NAME = parse_connection_string(AZURE_CONNECTION_STRING)["accountname"]`
causes
```python
#19 0.428   File "/usr/local/lib/python3.8/site-packages/azure/core/utils/_connection_string_parser.py", line 26, in parse_connection_string
#19 0.428     cs_args = [s.split("=", 1) for s in conn_str.strip().rstrip(";").split(";")]
#19 0.428 AttributeError: 'NoneType' object has no attribute 'strip'
```
During the collectstatic step when starting the container. So I'm tempted to leave this as separate for now.